api-reference/receipts/get-started.markdown

```diff
@@ -296,7 +296,7 @@ Successful POST requests will receive a response of 201 Created. The `Location`
 
 If you are not providing an image with your receipt data, the body of the request should be your receipt JSON:
 
-Receipt images may be posted along with data. In this case, Concur will use the provided image instead of generating a new one. To post data and an image, use multipart form data. The `Content-Type:multipart/form-data` header must be set. The image should be included under the key `image`, and the receipt JSON should be included under the key `receipt`.
+Receipt images may be posted along with data. In this case, Concur will use the provided image instead of generating a new one. To post data and an image, use multipart form data. The `Content-Type:multipart/form-data` header must be set. The image should be included under the key `image`, and the receipt JSON should be included under the key `receipt`. Image files are limited to a maximum size of 5MB. Accepted file formats for image files are: image/png, image/jpg, image/jpeg, image/tiff, image/tif,image/ gif, and application/pdf.
```

Comment:
Looks solid 👍  One thing I noticed is that there's a missing space between `image/tif` and `image/gif`, and an extra space inserted in `image/ gif`